plugins/source/azure/resources/services/costmanagement/views.go

```diff
@@ -36,19 +30,30 @@ func Views() *schema.Table {
 				},
 			},
 			{
-				Name:     "name",
-				Type:     schema.TypeString,
-				Resolver: schema.PathResolver("Name"),
-			},
-			{
-				Name:     "type",
+				Name:     "etag",
 				Type:     schema.TypeString,
-				Resolver: schema.PathResolver("Type"),
+				Resolver: schema.PathResolver("ETag"),
 			},
 		},
-
 		Relations: []*schema.Table{
 			view_queries(),
 		},
 	}
 }
+
+func fetchViews(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	cl := meta.(*client.Client)
+	svc, err := armcostmanagement.NewViewsClient(cl.Creds, cl.Options)
```

Comment:
This should be `pager := svc.NewListByScopePager("subscriptions/"+cl.SubscriptionId, nil)` otherwise no results will be returned.